release.yaml

```diff
@@ -2,18 +2,6 @@ elemental:
   - 106.0.0+up1.6.5
 elemental-crd:
   - 106.0.0+up1.6.5
-fleet:
-  - 104.1.2+up0.10.6
-  - 105.0.0+up0.11.0
```

Comment:
Why remove these 2 versions as well? Same remark about agent and CRDs charts.

Reply:
Fair enough. Thanks for clarifying.
As a side note, I could see on `dev-v2.11` that a few other charts (eg. `rancher-cis-benchmark`) also list multiple versions in `release.yaml`, so there may be a need to raise awareness about this org-wide.